lib/src/lib.nr

```diff
@@ -541,6 +549,31 @@ fn left_byte_shift<N>(input: [u8; N], n: u64) -> [u8; N]
     out
 }
 
+#[test]
+fn memcpy_test()
```

Comment:
Test names should start with test_. It's a convention. test_memcpy.

Reply:
Ahmad breaks this convention though

Reply:
Ok